### DISCUSSION TITLE
Open markdown file from CLI in preview mode

### DISCUSSION THREAD

**Discussion by cbochs** [zed-industries/zed#10379] [Feature Requests]
I know you can open a file at a specific line with the `zed` cli with:

```
zed README.md:111
```

I was wondering, however, the markdown file could be opened (by default) in "preview" mode instead of the editable document? In VSCode this can be accomplished with `workspace.editorAssociations`:

```json
  "workbench.editorAssociations": {
    "*.ipynb": "jupyter-notebook",
    "*.md": "vscode.markdown.preview.editor"
  },
```

**Comment by bitwizlabs** [comment]
I second this addition. The preview already works. Just needs a setting to open .md files that way by default.

{
  "languages": {
    "Markdown": {
      "open_in_preview": true
    }
  }
}

Simple addition. Would make Zed a real option for markdown-heavy workflows.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
